vehicle/mkvi/software/brakes/bspd.h

```diff
@@ -0,0 +1,38 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+//////////////////////// GPIO - DIGITAL ////////////////////////
+// These are digital outputs for avr-controlled LED outputs
+gpio_t MOTOR_5KW_LED = PB0;
+gpio_t BRAKE_LL_LED = PB1;
+gpio_t BSPD_TRIP_LED = PB2;
+
+// Monitor Pins connected to the logic-level (LL) side of the LSDs
+gpio_t BSPD_LL = PB3;
+gpio_t BRAKELIGHT_LL = PB4;
+
+// Digital sense pin for 5kW motor "on" state
+gpio_t MOTOR_CURRENT_SENSE = PB5;
+
+// Input for shutdown sense line
+// gpio_t BSPD_SHUTDOWN_SENSE = PC0;
```

Comment:
Ah, OK, so I've just caught something now that should have been caught in board review - what we're normally doing with the shutdown sense circuits is we literally want to feed a branch off of the shutdown circuit through the SS circuit to ensure there are no failure points measuring at some logic level point and the actual shutdown circuit. The shutdown sensing on your board takes the logic level as the input, which doesn't really make that distinction clear (and makes that circuit a bit redundant).

It's not the biggest of deals - as long as the relay works, which there isn't any reason to believe it won't, the values will be equivalent. Just wanted to make note of it here. May also be worth adding a comment as to why we're not using this value since that will be the convention elsewhere on the car for the sake of future readers.